### PR TITLE
Partner Program Announcement

### PR DESCRIPTION
**Pull request by zsarnett**
## Proposed change

Partner Program Announcement

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_posts/2022-07-12-partner-program.markdown [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+comments: true
+categories:
+- Announcements
+og_image: /images/blog/2022-07-12-partner-program/partner-program-logos.jpg
+---
+
+
+With Home Assistant, we integrate with over 1000 different APIs. The majority of these integrations are created and maintained by the Home Assistant community. Over the years a number of companies have stepped up to work with our community offering samples and engineering support. In a few cases, we saw companies pick up the maintenance of integrating their products in Home Assistant.
+
+Sadly, a couple of times it happened that companies went silent after their initial contribution, causing users to be wondering why new devices are not being supported. We want to protect our users from investing in products for their home that won't work well with Home Assistant.
```
Comment (frenck) [inline source/_posts/2022-07-12-partner-program.markdown]:
```suggestion
Sadly, a couple of times it happened that companies went silent after their initial contribution, causing users to be wondering why new devices are not being supported. We want to protect our users from investing in products for their homes that won't work well with Home Assistant.
```

File: source/_posts/2022-07-12-partner-program.markdown [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+Today we’re introducing the Works with Home Assistant program to allow manufacturers to show their support and commitment to Home Assistant and its community.
+
+The program requires manufacturers to maintain the integration of their products in Home Assistant, offer a good user experience, provide product samples and give us an engineering contact to escalate issues. In return, manufacturers will be able to use the “Works with Home Assistant” badge on their products and documentation. The terms of the Works with Home Assistant program are enforced in an agreement signed by both Nabu Casa and the manufacturer.
+
+<img src='/images/blog/2022-07-12-partner-program/partner-program-logos.jpg' alt="Decorative header." class='no-shadow'>
+
```
Comment (zsarnett) [inline source/_posts/2022-07-12-partner-program.markdown]:
```suggestion
<br>
```

Comment (frenck) [reply source/_posts/2022-07-12-partner-program.markdown]: This won't work here. Use HTML instead.

File: source/_posts/2022-07-12-partner-program.markdown [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+Sadly, a couple of times it happened that companies went silent after their initial contribution, causing users to be wondering why new devices are not being supported. We want to protect our users from investing in products for their homes that won't work well with Home Assistant.
+
+Today we’re introducing the Works with Home Assistant program to allow manufacturers to show their support and commitment to Home Assistant and its community.
+
+The program requires manufacturers to maintain the integration of their products in Home Assistant, offer a good user experience, provide product samples and give us an engineering contact to escalate issues. In return, manufacturers will be able to use the “Works with Home Assistant” badge on their products and documentation. The terms of the Works with Home Assistant program are enforced in an agreement signed by both Nabu Casa and the manufacturer.
+
+<img src='/images/blog/2022-07-12-partner-program/partner-program-logos.jpg' alt="Decorative header." class='no-shadow'>
+<br>
```
Comment (zsarnett) [inline source/_posts/2022-07-12-partner-program.markdown]:
```suggestion
<br><br>
```

**Review by frenck — APPROVED** [review]

**Pull request merged**